update!: Add Secureblue (#2961)
privacyguides/privacyguides.org: docs/desktop.md

For further information about how Qubes works, read our full [Qubes OS overview](os/qubes-overview.md) page.

### secureblue

<div class="admonition recommendation" markdown>

![Secureblue logo](assets/img/linux-desktop/secureblue.svg){ align=right }

> **Secureblue** is a security-hardened operating system based on [Fedora Atomic Desktop](https://fedoraproject.org/atomic-desktops/). It features their own Chromium-based browser inspired by GrapheneOS's Vanadium, [hardened_malloc](https://github.com/GrapheneOS/hardened_malloc), various kernel security improvements, per-network MAC address randomization, and many other [features](https://secureblue.dev/features).

Reviewer: ```suggestion
**Secureblue** is a security-focused operating system based on [Fedora Atomic Desktops](#fedora-atomic-desktops). It provides a [hardened memory allocator](https://github.com/GrapheneOS/hardened_malloc) and [Trivalent](https://github.com/secureblue/Trivalent), their own Chromium-based browser inspired by Vanadium for [GrapheneOS](android/distributions.md#grapheneos).
```

> [:octicons-home-16: Homepage](https://secureblue.dev/){ .md-button .md-button--primary }
> [:octicons-info-16:](https://secureblue.dev/install){ .card-link title=Documentation }
> [:octicons-code-16:](https://github.com/secureblue/secureblue){ .card-link title="Source Code" }
> [:octicons-heart-16:](https://secureblue.dev/donate){ .card-link title=Contribute }

Reviewer: ```suggestion
[:octicons-home-16: Homepage](https://secureblue.dev){ .md-button .md-button--primary }
[:octicons-info-16:](https://secureblue.dev/install){ .card-link title="Documentation" }
[:octicons-code-16:](https://github.com/secureblue/secureblue){ .card-link title="Source Code" }
[:octicons-heart-16:](https://secureblue.dev/donate){ .card-link title="Contribute" }
```

</details>

</div>

> 

Reviewer: ```suggestion

You can find a full list of the security [features](https://secureblue.dev/features) implemented in Secureblue on their website.

```
Splitting the part about Secureblue's features to the description under the recommendation box to follow this [guideline](https://github.com/privacyguides/privacyguides.org/pull/2789#discussion_r1819690660):

> stylistically I think the recommendation boxes shouldn't be overloaded with content (_especially_ technical content

For further information about how Qubes works, read our full [Qubes OS overview](os/qubes-overview.md) page.

### Secureblue

<div class="admonition recommendation" markdown>

![Secureblue logo](assets/img/linux-desktop/secureblue.svg){ align=right }

> **Secureblue** is a security-focused operating system based on [Fedora Atomic Desktops](#fedora-atomic-desktops). It includes a number of [security features](https://secureblue.dev/features) intended to proactively defend against the exploitation of both known and unknown vulnerabilities, and ships with [Trivalent](https://github.com/secureblue/Trivalent), their custom hardened web browser.

Reviewer 2: >their custom hardened web browser.

chromium*

[:octicons-home-16: Homepage](https://secureblue.dev){ .md-button .md-button--primary }
[:octicons-info-16:](https://secureblue.dev/install){ .card-link title="Documentation" }
[:octicons-code-16:](https://github.com/secureblue/secureblue){ .card-link title="Source Code" }
[:octicons-heart-16:](https://secureblue.dev/donate){ .card-link title="Contribute" }

</details>

</div>

> **Trivalent** is Secureblue's hardened Chromium for desktop Linux, inspired by [GrapheneOS](android/distributions.md#grapheneos)'s Vanadium browser.
> 
> Secureblue also provides GrapheneOS's [hardened memory allocator](https://github.com/GrapheneOS/hardened_malloc), and enables it globally (including for Flatpaks).

Reviewer: ```suggestion
**Trivalent** is Secureblue's hardened Chromium for desktop Linux inspired by [GrapheneOS](android/distributions.md#grapheneos)'s Vanadium browser.

Secureblue also provides GrapheneOS's [hardened memory allocator](https://github.com/GrapheneOS/hardened_malloc) and enables it globally (including for Flatpaks).
```
